### ISSUE TITLE
GUI (Inventory) builder

### ISSUE THREAD

**Issue by GuusLieben** [GuusLieben/DarwinServer#34]
Documentation for issue scheduled

**Comment by GuusLieben** [comment]
Removing self-assignment. If anyone wishes to pick up roadmap schedule this issue is open

**Comment by simbolduc** [comment]
Take inspiration of https://github.com/stefvanschie/IF/ and do a similar Panel system that uses x,y position in the chest and implementing PaginatedPane.

**Comment by GuusLieben** [comment]
Would XML/JSON/YML importing be a desired feature as well @simbolduc?

**Comment by simbolduc** [comment]
> Would XML/JSON/YML importing be a desired feature as well @simbolduc? 

I don't think that would be so useful other than static GUI such as /menu. Maybe later in the making, but definitly not an important feature to me.

**Comment by simbolduc** [comment]
Maybe having a way to set borders of glass panes or other material.

**Comment by GuusLieben** [comment]
Will update the roadmap here once a plan is constructed. Bordered panes could be similar to [IF's OutlinePane](https://github.com/stefvanschie/IF/wiki/Outline-Pane). Some default pane implementations I think should be added (besides a normal, static pane) are:
- Pagination pane
- Bordered pane
- Auto alignment pane (Top/bottom, left/right, center)
- Abstract pane to be extended by developers
